lib/msf/core/rpc/v10/rpc_db.rb

```diff
@@ -791,6 +791,12 @@ def rpc_analyze_host(xopts)
         {
           mtype: mod.type,
           mname: mod.fullname,
+          state: result.to_s,
```

Comment:
As noted in description, what are thought on making this ENUM value and adding a separate descriptive string?

```suggestion
          state: result.state,
          detail: result.to_s,
```

Reply:
Adding an enum sounds good to me :+1:

if i remember correctly this string gets output to the console though, if that's the case we might want to map it in the console command to human readable strings

Reply:
Do you think keeping `.to_s` as the `human readable message` and a new method/accessor for `state` would be reasonable?

Adding the ENUM would then leave the CLI alone and just enhance API based interaction.